### PR TITLE
Binding desc non jj 2/2

### PR DESCRIPTION
**Pull request by ElianaDalSasso**
Dear all, here are my last four binding descriptions of non-Juel-Jensen manuscripts! :)

### CODE REVIEW

**Review by thea-m — APPROVED** [review]
Great, thank you! One minor remark (more stylistic).

File: OxfordBodleian/BodNewMSS/BDLaethg46.xml [hunk 1]
```diff
@@ -125,39 +128,48 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+
+                        <bindingDesc>
+                              <binding contemporary="true Ethiopian">
+                                <p>The binding belongs to the Ethiopian binding tradition.</p>
+                                  <decoNote xml:id="b1" type="bindingMaterial">
+                                    The binding is enclosed in a <material key="textile">textile</material> chemise furnished with an external <material key="leather">leather</material>
+                                    layer.
+                                  </decoNote>
+                                  <decoNote xml:id="b2" type="Cover">
+                                    The binding is protected by a coarse chemise of neutral-coloured fabric externally covered by a thick leather layer sewn to its borders.
```
[the remark this answers is not in the document]
Comment (ElianaDalSasso) [reply OxfordBodleian/BodNewMSS/BDLaethg46.xml:154]: You are right! I'll integrate the information in b1 and suppress b2. Thank you!

**Review by MassimoVilla — APPROVED** [review]
Thank you Eliana. No comments on my part

File: OxfordBodleian/Ullendorff/BDLeu4.xml [hunk 2]
```diff
@@ -173,37 +173,78 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                            <binding xml:id="binding">
-                                <decoNote xml:id="b1"></decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">
-                           <material key="wood"/>
-                        </decoNote>
+
+                        <bindingDesc>
+                            <binding xml:id="binding" contemporary="true Ethiopian">
+                              <p>
+                                The manuscript was repaired several times, as testify the various threads visible on the spine and holes drilled through the boards.
```
Comment (gnizla) [inline OxfordBodleian/Ullendorff/BDLeu4.xml]: I would say "as testified by the various threads visible on the spine and holes drilled through the boards." But you can change this when we do the editing

**Review by gnizla — APPROVED** [review]

**Comment by gnizla** [comment]
Thanks a lot for this @ElianaDalSasso !

**Comment by ElianaDalSasso** [comment]
Thank you @MassimoVilla! Sorry, I forgot to do it yesterday!

**Pull request merged**